zebra-state/src/service/finalized_state/disk_format.rs

```diff
@@ -259,12 +259,18 @@ impl DiskSerialize for rocksdb::WriteBatch {
 /// Helper trait for retrieving values from rocksdb column familys with a consistently
 /// defined format
 pub trait DiskDeserialize {
-    /// Serialize the given key and use that to get and deserialize the
+    /// Serialize the given key and use it to get and deserialize the
     /// corresponding value from a rocksdb column family, if it is present.
```

Comment:
I'm not sure serialization is really that important here?
```suggestion
    /// Returns the value for `key` in the rocksdb column family `cf`, if present.
```